ui/service.go

```diff
@@ -65,17 +67,27 @@ func New(things mainflux.ThingsServiceClient, sdk sdk.SDK) Service {
 	}
 }
 
+func parseTemplate(tmps ...string) (tpl *template.Template, err error) {
+	for _, tmp := range tmps {
+
+		tpl = template.New(tmp)
+		tpl = tpl.Funcs(template.FuncMap{
+			"toJSON": func(data map[string]interface{}) string {
+				ret, _ := json.Marshal(data)
+				return string(ret)
+			},
+		})
+		a := append(tmplFiles, templateDir+tmp)
+		tpl, err = tpl.ParseFiles(a...)
```

Comment:
This should not be in `range` loop, but later, in the end, when all files are well prepared in the array.